Cubical/HITs/Pushout/Properties.agda

```diff
@@ -0,0 +1,209 @@
+{-# OPTIONS --cubical --safe #-}
+
```

Comment:
You can add your name in a comment up here if you want so that the authorship is preserved